airbyte-webapp/src/components/ui/NextTable/NextTable.module.scss

```diff
@@ -46,7 +46,7 @@
   }
 
   &:first-child {
-    padding-left: calc(variables.$spacing-2xl + variables.$spacing-sm);
+    padding-left: calc(variables.$spacing-xl + variables.$spacing-sm);
```

Comment:
I made these changes in this file because the padding on the first child seemed weirdly high to me, and the custom padding in the cell data didn't seem necessary and just resulted in misaligned columns. Here are some screenshots showing what this changes in various tables in the app:

### Sources page
before:
<img width="947" alt="Screen Shot 2023-02-22 at 6 34 03 PM" src="https://user-images.githubusercontent.com/22731524/220809199-ca4c22fb-d777-4c62-a0a3-74df106dfbf5.png">

after:
<img width="951" alt="Screen Shot 2023-02-22 at 6 34 12 PM" src="https://user-images.githubusercontent.com/22731524/220809224-58041483-9653-4657-8884-29fcb0441e68.png">

### Connections page
before:
<img width="956" alt="Screen Shot 2023-02-22 at 6 35 09 PM" src="https://user-images.githubusercontent.com/22731524/220809357-8c20f71f-d130-42da-8f1c-97bbdde8b905.png">

after:
<img width="953" alt="Screen Shot 2023-02-22 at 6 35 28 PM" src="https://user-images.githubusercontent.com/22731524/220809372-b687900d-4c35-4d07-8693-4fbdb8c2bc1e.png">

### Settings > Sources page
before:
<img width="935" alt="Screen Shot 2023-02-22 at 6 36 24 PM" src="https://user-images.githubusercontent.com/22731524/220809526-0a1a026c-281f-49d2-b6f5-31bb4e8f3210.png">

after:
<img width="941" alt="Screen Shot 2023-02-22 at 6 36 42 PM" src="https://user-images.githubusercontent.com/22731524/220809539-61ff44e6-bd78-474b-bb40-bcba3b26d19e.png">

I think the weird padding rules in these table styles can still be cleaned up more, but I didn't want to spend too much time on that for this PR specifically. Lmk what you think

Reply:
It looks like the first column has a left padding of 32px, set here: https://github.com/airbytehq/airbyte-platform/blob/c6d8f28eafafeceaca7016f3ba0d6caa120172ab/airbyte-webapp/src/components/EntityTable/ImplementationTable.module.scss#L3

However setting the padding left of this to 32px as well doesn't work, because the header text is wrapped in a button and that has a default of 6px by the browser stylesheet:
<img width="737" alt="Screenshot 2023-02-23 at 11 54 49" src="https://user-images.githubusercontent.com/1508364/220887091-e931fbf0-c093-46d2-a3ea-2739dd4af252.png">


However, even removing this doesn't fix it everywhere because the connection table overwrites the 32px left padding... No easy win here. FYI @edmundito @josephkmh we should align these padding values for all tables for a consistent look.

Reply:
Yeah these tables are really due for an overhaul. IIRC `EntityTable` is set to be replaced by `NextTable`, though I'm not sure on the timeline there, and whether any team has replacing these specific tables on their roadmap. @edmundito or @dizel852 do you have more insight here?

Reply:
Okay, sounds like I should go ahead and undo the `NextTable.module.scss` changes I made in this PR and instead just change it for the specific table I'm adding here